docker-compose.yml

```diff
@@ -1,68 +1,90 @@
 # For more information: https://laravel.com/docs/sail
-version: '3'
+version: "3"
 services:
     laravel.test:
         build:
-            context: ./vendor/laravel/sail/runtimes/8.0
+            context: ./vendor/laravel/sail/runtimes/8.3
             dockerfile: Dockerfile
             args:
-                WWWGROUP: '${WWWGROUP}'
-        image: sail-8.0/app
+                WWWGROUP: "${WWWGROUP}"
+        image: sail-8.3/app
+        extra_hosts:
+            - "host.docker.internal:host-gateway"
         ports:
-            - '${APP_PORT:-80}:80'
+            - "${APP_PORT:-80}:80"
+            - "${VITE_PORT:-5173}:${VITE_PORT:-5173}"
         environment:
-            WWWUSER: '${WWWUSER}'
+            WWWUSER: "${WWWUSER}"
             LARAVEL_SAIL: 1
-            XDEBUG_MODE: '${SAIL_XDEBUG_MODE:-off}'
-            XDEBUG_CONFIG: '${SAIL_XDEBUG_CONFIG:-client_host=host.docker.internal}'
+            XDEBUG_MODE: "${SAIL_XDEBUG_MODE:-off}"
+            XDEBUG_CONFIG: "${SAIL_XDEBUG_CONFIG:-client_host=host.docker.internal}"
+            IGNITION_LOCAL_SITES_PATH: "${PWD}"
         volumes:
-            - '.:/var/www/html'
+            - ".:/var/www/html"
         networks:
             - sail
-        depends_on:
-            - mysql
-            - redis
+    # laravel.test:
+    #     build:
+    #         context: ./vendor/laravel/sail/runtimes/8.3
+    #         dockerfile: Dockerfile
+    #         args:
+    #             WWWGROUP: "${WWWGROUP}"
+    #     image: sail-8.0/app
+    #     ports:
+    #         - "${APP_PORT:-80}:80"
+    #     environment:
+    #         WWWUSER: "${WWWUSER}"
+    #         LARAVEL_SAIL: 1
+    #         XDEBUG_MODE: "${SAIL_XDEBUG_MODE:-off}"
+    #         XDEBUG_CONFIG: "${SAIL_XDEBUG_CONFIG:-client_host=host.docker.internal}"
+    #     volumes:
+    #         - ".:/var/www/html"
+    #     networks:
+    #         - sail
+    #     depends_on:
+    #         - mysql
+    #         - redis
     mysql:
-        image: 'mariadb'
+        image: "mariadb"
         ports:
-            - '${FORWARD_DB_PORT:-3306}:3306'
+            - "${FORWARD_DB_PORT:-3306}:3306"
         environment:
-            MYSQL_ROOT_PASSWORD: '${DB_PASSWORD}'
-            MYSQL_DATABASE: '${DB_DATABASE}'
-            MYSQL_USER: '${DB_USERNAME}'
-            MYSQL_PASSWORD: '${DB_PASSWORD}'
-            MYSQL_ALLOW_EMPTY_PASSWORD: 'yes'
+            MYSQL_ROOT_PASSWORD: "${DB_PASSWORD}"
+            MYSQL_DATABASE: "${DB_DATABASE}"
+            MYSQL_USER: "${DB_USERNAME}"
+            MYSQL_PASSWORD: "${DB_PASSWORD}"
+            MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
         volumes:
-            - 'sailmysql:/var/lib/mysql'
+            - "sailmysql:/var/lib/mysql"
```

Comment:
We should change the volume and in general name them unique for our projects, as otherwise we could cause some issues for people already having outdated volumes